From 75e707331913dde42fa624abe4248f5b9fc43c0a Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Tue, 5 May 2015 14:34:40 +0800
Subject: [PATCH] MLK-10818-3 ARM: dts: add audio route configuration for
 imx7d-sdb board

add audio route configuration for imx7d-sdb board.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index e45f53e..c2fd1a8 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -97,6 +97,9 @@
 		codec-master;
 		/* JD2: hp detect high for headphone*/
 		hp-det = <2 0>;
+		audio-routing =
+			"LINPUT1", "Main MIC",
+			"Main MIC", "MICB";
 	};
 
 	sound-hdmi {
-- 
1.8.0

